Application/EdFi.Ods.AdminApi.AdminConsole/Features/Healthcheck/AddHealthCheck.cs

```diff
@@ -0,0 +1,79 @@
+// SPDX-License-Identifier: Apache-2.0
+// Licensed to the Ed-Fi Alliance under one or more agreements.
+// The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+// See the LICENSE and NOTICES files in the project root for more information.
+
+using System.ComponentModel.DataAnnotations;
+using EdFi.Ods.AdminApi.AdminConsole.Infrastructure.Services.HealthChecks.Commands;
+using EdFi.Ods.AdminApi.Common.Features;
+using EdFi.Ods.AdminApi.Common.Infrastructure;
+using FluentValidation;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Routing;
+using Newtonsoft.Json;
+using Swashbuckle.AspNetCore.Annotations;
+
+namespace EdFi.Ods.AdminApi.AdminConsole.Features.Healthcheck;
+public class AddHealthCheck : IFeature
+{
+    public void MapEndpoints(IEndpointRouteBuilder endpoints)
+    {
+        AdminApiEndpointBuilder.MapPost(endpoints, "/healthcheck", Execute)
```

Comment:
Is this actually going to be `GET /adminconsole/healthcheck`? Or will it be `GET /healthcheck`?